src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public int getRandomBall() {
+        java.util.Random random = new Random();
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomBall() method